From 5e093b9568891cda78f080cf2aaaf38661a4d2c1 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Wed, 28 Mar 2018 18:52:58 -0400
Subject: [PATCH] Unbreak config watcher in the main process

---
 src/path-watcher.js | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/src/path-watcher.js b/src/path-watcher.js
index ff7e8fd56..44a59dec7 100644
--- a/src/path-watcher.js
+++ b/src/path-watcher.js
@@ -176,7 +176,7 @@ class NativeWatcher {
     this.onEvents = this.onEvents.bind(this)
     this.onError = this.onError.bind(this)
 
-    this.subs.add(atom.config.onDidChange('core.fileSystemWatcher', async () => {
+    this.subs.add(this.getConfig().onDidChange('core.fileSystemWatcher', async () => {
       if (this.state === WATCHER_STATE.STARTING) {
         // Wait for this watcher to finish starting.
         await new Promise(resolve => {
@@ -197,9 +197,18 @@ class NativeWatcher {
     }))
   }
 
+  // Private: get config instance for the current application
+  getConfig() {
+    if (typeof(global.atom) != 'undefined') {
+      return global.atom.config
+    } else {
+      return global.atomApplication.config
+    }
+  }
+
   // Private: Read the `core.fileSystemWatcher` setting to determine the filesystem backend to use.
   getCurrentBackend () {
-    const setting = atom.config.get('core.fileSystemWatcher')
+    const setting = this.getConfig().get('core.fileSystemWatcher')
     return BACKENDS[setting] || DEFAULT_BACKEND
   }
 
-- 
2.16.1

