From 3a644b25a9d53496de4960afd9fa4afed95ddae7 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Thu, 5 May 2016 17:23:35 -0400
Subject: [PATCH] Unbundle electron

---
 atom.sh | 18 +++++-------------
 1 file changed, 5 insertions(+), 13 deletions(-)

diff --git a/atom.sh b/atom.sh
index cd28dd334..2fa1f436b 100755
--- a/atom.sh
+++ b/atom.sh
@@ -107,17 +107,9 @@ elif [ $OS == 'Linux' ]; then
   SCRIPT=$(readlink -f "$0")
   USR_DIRECTORY=$(readlink -f $(dirname $SCRIPT)/..)
 
-  case $CHANNEL in
-    beta)
-      ATOM_PATH="$USR_DIRECTORY/share/atom-beta/atom"
-      ;;
-    dev)
-      ATOM_PATH="$USR_DIRECTORY/share/atom-dev/atom"
-      ;;
-    *)
-      ATOM_PATH="$USR_DIRECTORY/share/atom/atom"
-      ;;
-  esac
+  export NPM_CONFIG_NODEDIR="{{NPM_CONFIG_NODEDIR}}"
+  export ATOM_RESOURCE_PATH="{{ATOM_RESOURCE_PATH}}"
+  ATOM_PATH="{{ATOM_PATH}}"
 
   ATOM_HOME="${ATOM_HOME:-$HOME/.atom}"
   mkdir -p "$ATOM_HOME"
@@ -127,11 +119,11 @@ elif [ $OS == 'Linux' ]; then
   [ -x "$ATOM_PATH" ] || ATOM_PATH="$TMPDIR/atom-build/Atom/atom"
 
   if [ $EXPECT_OUTPUT ]; then
-    "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@"
+    "$ATOM_PATH" --app="$ATOM_RESOURCE_PATH" --executed-from="$(pwd)" --pid=$$ "$@"
     exit $?
   else
     (
-    nohup "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@" > "$ATOM_HOME/nohup.out" 2>&1
+    nohup "$ATOM_PATH" --app="$ATOM_RESOURCE_PATH" --executed-from="$(pwd)" --pid=$$ "$@" > "$ATOM_HOME/nohup.out" 2>&1
     if [ $? -ne 0 ]; then
       cat "$ATOM_HOME/nohup.out"
       exit $?
-- 
2.16.1

