From 6df8fc485ac80dd9e93c0cbc46ed0c0aa323ad53 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Wed, 10 Feb 2016 14:45:13 -0500
Subject: [PATCH] Parametrize v8.gyp location

---
 node.gyp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/node.gyp b/node.gyp
index d431210..fea5e07 100644
--- a/node.gyp
+++ b/node.gyp
@@ -1,6 +1,7 @@
 {
   'variables': {
     'v8_use_snapshot%': 'false',
+    'v8_gyp_path%': 'deps/v8/tools/gyp/v8.gyp',
     'node_use_dtrace%': 'false',
     'node_use_lttng%': 'false',
     'node_use_etw%': 'false',
@@ -89,8 +90,8 @@
       'dependencies': [
         'node_js2c#host',
         'deps/cares/cares.gyp:cares',
-        # 'deps/v8/tools/gyp/v8.gyp:v8',
-        # 'deps/v8/tools/gyp/v8.gyp:v8_libplatform'
+        '<(v8_gyp_path):v8',
+        '<(v8_gyp_path):v8_libplatform'
       ],
 
       'include_dirs': [
-- 
2.4.10

