From 9b7abffc141f5b134164dda59759ba769ee5cc85 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <el@prans.net>
Date: Mon, 7 Jun 2021 12:24:22 -0700
Subject: [PATCH] Fix node openssl_fips declaration

---
 common.gypi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common.gypi b/common.gypi
index cebf2121ff..922c134eb2 100644
--- a/common.gypi
+++ b/common.gypi
@@ -27,7 +27,7 @@
 
     'clang%': 0,
 
-    'openssl_fips%': '',
+    'openssl_fips': '',
 
     # Don't use ICU data file (icudtl.dat) from V8, we use our own.
     'icu_use_data_file_flag%': 0,
-- 
2.31.1

